runtime/cudaq/platform/default/rest/RemoteRESTQPU.cpp

```diff
@@ -151,6 +151,16 @@ class RemoteRESTQPU : public cudaq::QPU {
   void clearShots() override { nShots = std::nullopt; }
   virtual bool isRemote() override { return !emulate; }
 
+  /// @brief Set the noise model, only allow this for
+  /// emulation.
+  void setNoiseModel(cudaq::noise_model *model) override {
+    if (!emulate)
```

Comment:
How would you emulate and run on the noisy simulator?

Reply:
We might have to add a flag to the driver, but in principle you could do it by just updating the simulation library being linked to `libnvqir-density-matrix-cpu.so`